ArduCopter/Parameters.cpp

```diff
@@ -918,6 +918,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("TUNE_MAX", 32, ParametersG2, tuning_max, 0),
 
+    // @Param: RTL_TYPE
+    // @DisplayName: RTL handling type
+    // @Description: This controls how RTL is handled. A value of zero means a conventional RTL. A value of 1 uses the closest DO_LAND_START marker in the mission waypoints to fly a pre-planned path. If no DO_LAND_START markers a found a conventional RTL is used.
```

Comment:
Should note that this is using the closest geographic DO_LAND_START to the current vehicle's position not the closest mission planned number.